js/preload/preloadMain.js

```diff
@@ -28,6 +28,23 @@ const KeyCodes = {
     Esc: 27,
 };
 
+let Search;
+let SearchUtils;
+
+try {
+    Search = remote.require('swift-search').Search;
+} catch (e) {
+    // eslint-disable-next-line no-console
+    console.warn("Failed to initializing Search");
+}
+
+try {
+    SearchUtils = remote.require('swift-search').SearchUtils;
+} catch (e) {
+    // eslint-disable-next-line no-console
+    console.warn("Failed to initializing SearchUtils");
```

Comment:
Please change this to `Failed to initialize swift search. You'll need to include the search dependency. Contact the developers for more details`